lib/manager/pip-compile/artifacts.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { exec as _exec } from 'child_process';
```

Comment:
Don't we have it in exec util? 🤔

Reply:
This file was copy/pasted from `pipenv` so I prefer to leave this stuff as-is and refactor/fix it later at the same time as pipenv's.